packaging/common/postinst

```diff
@@ -23,9 +23,15 @@ case $1 in
   fi
 
   # Configure OpenSSL to use the engine
-  if [ -x /usr/lib/keysinuse/keysinuseutil ]; then
+  if [ -x /usr/bin/keysinuseutil ]; then
     echo "Enabling keysinuse engine for OpenSSL"
-    /usr/lib/keysinuse/keysinuseutil -update-default install|| echo "Configuring engine failed"
+    /usr/bin/keysinuseutil -update-default install || echo "Configuring engine failed"
+  fi
+
+  # Config used to reside in /usr/lib/keysinuse. Create symlink for
+  # backwards compat and discoverability
+  if [ -e /etc/keysinuse/keysinuse.cnf ]; then
+    ln -s /etc/keysinuse/keysinuse.cnf /usr/lib/keysinuse/keysinuse.cnf
```

Comment:
>ln -s /etc/keysinuse/keysinuse.cnf /usr/lib/keysinuse/keysinuse.cnf [](http://example.com/codeflow?start=4&length=67)

What's the scenario we're concerned about here? Someone on a machine using the old version upgrades to the new version? In that case you're just overwriting their existing config file with a symlink. I don't see the benefit here.

Reply:
This is just to give us a temporary buffer as we upgrade existing deployments. Our guidance and deployment model has been to use '.include' in the default OpenSSL config, and point to this location. We should be safe to remove this, but I'd like to keep this as a failsafe when we upgrade and then remove in 0.3.2